.github/workflows/release.yaml

```diff
@@ -149,4 +149,33 @@ jobs:
           name: v${{ env.VERSION }}
           draft: true
           prerelease: ${{ startsWith(env.VERSION, '0') }}
-          files: artifacts/*
\ No newline at end of file
+          files: artifacts/*
+
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    needs: [ prepare ]
+    env:
+      VERSION: ${{ needs.prepare.outputs.version }}
```

Comment:
This will be the CLI's version, not the registry's.

Reply:
Good eyes, I will fix this.